openedx/features/survey_report/models.py

```diff
@@ -35,3 +35,31 @@ class SurveyReport(models.Model):
     class Meta:
         ordering = ["-created_at"]
         get_latest_by = 'created_at'
+
+
+class SurveyReportUpload(models.Model):
+    """
+    This models stores information about send request of a survey report.
+
+    .. no_pii:
+
+    fields:
+    - sent_at: Date when the report was sent.
+    - report: The report that was sent.
+    - status: Request status code.
+    - request_details: Information about the send request.
+    """
+    sent_at = models.DateTimeField(auto_now=True, help_text="Date when the report was sent.")
+    report = models.ForeignKey(SurveyReport, on_delete=models.CASCADE, help_text="The report that was sent.")
+    status = models.IntegerField(help_text="Request status code.")
+    request_details = models.CharField(
+        max_length=255,
+        null=True,
+        blank=True,
+        help_text="Information about the send request."
+    )
+
+    def is_uploaded(self) -> bool:
+        response = Response()
+        response.status_code = self.status
+        return response.ok
```

Comment:
```suggestion
        return self.status == HTTP_200_OK
```

Reply:
response.ok is different to HTTP_200_OK, I decided use it because with that we won't have problems if we receive a different success status.

Reply:
Creating a response object is the best way of doing it -just asking-?

Reply:
FWIW, I made the same assumption as @mariajgrimaldi, that `response.ok` was equivalent to a 200 return code. If your intention is to accept all 2xx response codes, you could implement this as:

`return 200 <= self.status < 300`

That might make the intention clearer to those who aren't familiar with the library.

Reply:
@mariajgrimaldi I don't any know other way to do it.

Reply:
why don't you try Dave's suggestion?

Reply:
@mariajgrimaldi @ormsbee Done.